src/main/java/jun/yu/code/coverage/code/review/FileOutputDemo.java

```diff
@@ -0,0 +1,32 @@
+package jun.yu.code.coverage.code.review;
+
+import java.io.*;
+
+class FileOutputDemo 
+{   
+
+
+        public static void main(String[] args) throws IOException
+        {              
+                FileOutputStream out; // declare a file output object
+                PrintStream p; // declare a print stream object
+
+                try
+                {
+                        // Create a new file output stream
+                        // connected to "myfile.txt"
+                        out = new FileOutputStream("myfile.txt");
+
+                        // Connect print stream to the output stream
+                        p = new PrintStream( out );
+        
+                        p.println ("This is written to a file");
+
+                        p.close();
+                }
+                catch (Exception e)
+                {
+                    //throw new IOException(e);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)